src/dmd/globals.d

```diff
@@ -27,12 +27,15 @@ template xversion(string s)
 
 enum IN_GCC     = xversion!`IN_GCC`;
 
-enum TARGET_LINUX   = xversion!`linux`;
-enum TARGET_OSX     = xversion!`OSX`;
-enum TARGET_FREEBSD = xversion!`FreeBSD`;
-enum TARGET_OPENBSD = xversion!`OpenBSD`;
-enum TARGET_SOLARIS = xversion!`Solaris`;
-enum TARGET_WINDOS  = xversion!`Windows`;
+enum TARGET
```

Comment:
Perhaps a nested enum: `Target.Kind`?

Reply:
Hmm. As @ibuclaw mentioned `Target` needs to be refactored anyways and at the moment it would be rather empty:

```
struct Target
{
    enum Kind { ...}
}


Also I recall that @ZombineDev wanted to introduce TargetTriplets.
-> I think we can do this change once the time comes?

Reply:
> I think we can do this change once the time comes?

Ok.